### PR TITLE
Vmoutput maps

### PR DESCRIPTION
**Pull request by camilbancioiu**
* Convert `VMOutput.OutputAccounts` into a map
* Convert `OutputAccount.StorageUpdates` into a map

### CODE REVIEW

File: output.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type OutputAccount struct {
 	// that should be updated in the account storage.
 	// Please note that it is likely that not all existing
 	// account storage keys show up here.
```
Comment (andrei-marinica) [inline output.go]: Could you please also add some documentation in the comments, as to what the map keys represent?

File: output.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ type VMOutput struct {
 	// This data tells the network how to update the account data.
 	// It can contain new accounts or existing changed accounts.
 	// Note: the current implementation might also retrieve accounts that were not changed.
```
Comment (andrei-marinica) [inline output.go]: I'm guessing here the map key is the account address, right?

File: output.go (hunk 1 above)
Comment (camilbancioiu) [reply output.go]: Updated the comment.

File: output.go (hunk 2 above)
Comment (camilbancioiu) [reply output.go]: Indeed. The comment is now up to date.

**Pull request merged**